src/dotnet-install.ps1

```diff
@@ -131,6 +131,29 @@ function Say($str) {
     }
 }
 
+function Say-Warning($str) {
+    try
+    {
+        Write-Warning "dotnet-install: $str"
+    }
+    catch
+    {
+        # Some platforms cannot utilize Write-Warning (Azure Functions, for instance). Fall back to Write-Output
+        Write-Output "Warning: dotnet-install: $str"
```

Comment:
Not sure how output of `Write-Warning` looks like, however I would expect dotnet-install always be first.
```
dotnet-install: my output message
dotnet-install: Warning: my warning message 
dotnet-install: Error: my error message 
```